jobs/geo/write_building_footprints.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from jobs.utils.snowflake import snowflake_connection_from_environment
+
+
+def write_building_footprints(conn):
+    """Grab Microsoft state building footprint data for California from Snowflake and write to an S3 bucket."""
+    import geopandas
+    import shapely
+
+    sql_transform = """
```

Comment:
This isn't really a transform, it's configuring session parameters

Reply:
Will name it better!